tools/isobuild/bundler.js

```diff
@@ -184,6 +185,62 @@ exports.ignoreFiles = [
       */
 ];
 
+const METEOR_IGNORE = '.meteorignore'
+
+/*
+    .meteorignore works *like*
+    https://git-scm.com/docs/gitignore
+    what's implemented: 
+      - blank line separator for readability
+      - A line starting with # as a comment, \# or [#] is not comment.
+      - Trailing spaces are ignored.
+      - glob stars ** matches all directory. 
+        for example trailing /** matches eveying inside   */   
+//       a/**/b matches a/b a/x/b, a/x/y/b...  
+/*
+    what's implemented but not in the specification.
+      - ? for match single character
+      - [] to match character group.
+
+    What's not effective: (doesn't work.)
+      - ! negative pattern won't be effective unless the below known issue is resolved.
+
+    known issue:
+      ignoreFiles take regExp list with default logical *or*, 
+      instead of glob pattern list with optional logical *and* for negative pattern.
+        *        # ignore all files
+        !*.html  # but (AND) not ignore *.html, ignore files that matches * AND !*.html
+        !*.css   # (AND) not ignore *.css
+        !*.js    # (AND) not ignore *.js
+*/
+
+if (files.exists(METEOR_IGNORE)) {
+  var dotIgnoreFiles = files.readFile(METEOR_IGNORE).toString('utf8')
+  exports.ignoreFiles = exports.ignoreFiles.concat( // wanted use array spread operator .push(...dotIgnoreFiles.split( maybe from v1.6
+    dotIgnoreFiles
```

Comment:
Take my comment lightly as I am not a contributor here.  My intent is just to encourage meteor to support the community standard features around ignoring files.  If that is already done by your filter, then I'm very happy.

No further concerns here :)